lib/features/family/features/home_screen/widgets/missions_container.dart

```diff
@@ -43,30 +46,32 @@ class MissionsContainer extends StatelessWidget {
       onTap: () => context.pushNamed(FamilyPages.missions.name),
       child: Stack(
         children: [
-          Column(
-            children: [
-              const Row(),
-              const SizedBox(height: 16),
-              const FaIcon(FontAwesomeIcons.bolt,
-                  color: FamilyAppTheme.primary20),
-              const SizedBox(height: 12),
-              TitleSmallText(
-                missionStats!.missionsToBeCompleted > 0
-                    ? 'Missions available'
-                    : 'All missions completed!',
-              ),
-              const SizedBox(height: 4),
-              if (missionStats!.missionsToBeCompleted > 0)
-                BodySmallText(
-                  '${missionStats!.missionsToBeCompleted} missions to be completed',
+          Center(
```

Comment:
Should this make use of the FunGoalCard widget? :)

Reply:
I think no, maybe we can discuss this at 2, but shoehorning everything into the same "Card" widget is gonna cause problems later down the line in my opinion.

Reply:
@TammiLion  But its a design system element, right? :)

Reply:
we had a huddle and it is clear to me now, and now i think it indeed should also be the same component, lets fix that in a different pr though! right now the fungoalcard widget isn't good enough

Reply:
👌